Rubberduck.Refactorings/IRefactoringPresenterFactory.cs

```diff
@@ -3,8 +3,8 @@
     public interface IRefactoringPresenterFactory
     {
         TPresenter Create<TPresenter, TModel>(TModel model)
-            where TPresenter : class
-            where TModel : class;
+            where TPresenter : class, IRefactoringPresenter<TModel>
+            where TModel : class, IRefactoringModel;
```

Comment:
I think `class` is now redundant now that we have an interface to restrict the type allowed.

Reply:
The `class` is required. Otherwise, the compiler complains that certain operations are only valid for reference types.